contracts/FlowServiceAccount.cdc

```diff
@@ -187,6 +188,11 @@ access(all) contract FlowServiceAccount {
         }
     }
 
+    access(all)
```

Comment:
This should not be access all in my view, should be only callable by the service account